sps30-uart/sps30.c

```diff
@@ -46,6 +46,7 @@
 #define SPS30_CMD_DEV_INFO 0xd0
 #define SPS30_CMD_DEV_INFO_SUBCMD_GET_SERIAL                                   \
     { 0x03 }
+#define SPS30_CMD_READ_VERSION 0xD1
```

Comment:
note on consistency: the others use lower case hex...

Reply:
Fixed.